schemas/exchange/min-meta/json-min-db-pdbx_core-col-pdbx_core_branched_entity_instance.json

```diff
@@ -1355,6 +1355,38 @@
                      }
                   ]
                },
+               "connect_type": {
```

Comment:
I didn't know that we calculate it for sugars too. Are they added as ligands for polymers? What's in "rcsb_ligand_neighbors" for branched entities?

Reply:
Branched instances are treated like polymers, I believe, so the interactions are between small ligands and the branched entity (e.g., [60FD.J](https://data.rcsb.org/graphql/index.html?query=%7B%0A%20%20branched_entity_instance(entry_id%3A%20%226OFD%22%2C%20asym_id%3A%20%22J%22)%20%7B%0A%09%09%09rcsb_ligand_neighbors%20%7B%0A%09%09%09%20%20alt_id%0A%09%09%09%20%20atom_id%0A%09%09%09%20%20auth_seq_id%0A%09%09%09%20%20comp_id%0A%09%09%09%20%20distance%0A%09%09%09%20%20ligand_alt_id%0A%09%09%09%20%20ligand_asym_id%0A%09%09%09%20%20ligand_atom_id%0A%09%09%09%20%20ligand_comp_id%0A%09%09%09%20%20ligand_entity_id%0A%09%09%09%20%20ligand_is_bound%0A%09%09%09%20%20ligand_model_id%0A%09%09%09%20%20seq_id%0A%09%09%09%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A)).

However, I did not include these in the feature positions, since they were presenting issues with the sequence ID I believe.